Vagrantfile

```diff
@@ -75,7 +75,7 @@ DOCKERCFG = File.expand_path(ENV['DOCKERCFG'] || "~/.dockercfg")
 
 DOCKER_OPTIONS = ENV['DOCKER_OPTIONS'] || ''
 
-KUBERNETES_VERSION = ENV['KUBERNETES_VERSION'] || '1.6.2'
+KUBERNETES_VERSION = ENV['KUBERNETES_VERSION'] || '1.6.4'
```

Comment:
This shouldn't be part of this PR since it's not related to the upgrade to `etcd3`.